### ISSUE TITLE
Add `Momentum.afterNextAnimation`, similar to `Tracker.afterFlush`

### ISSUE THREAD

**Issue by tmeasday** [percolatestudio/meteor-momentum#22]
Would probably require the reintroduction of something like #13, but that's been mooted anyway #17.

@zol What do you think?

**Comment by zol** [comment]
Yeah, There is certainly a good case for a more full featured 'plugin manager' that probably looks like a combination of #13 + #17. I don't think that's trivial though (#17 in particular) so we'd need to justify and schedule that case for it.

**Comment by tmeasday** [comment]
Yeah i don't think any of it's trivial. Maybe it's another package?
